### PR TITLE
Logo now points to current environment

### PR DESCRIPTION
**Pull request by edwoodward**
Dependent on https://github.com/openstax/tutor-deployment/pull/553

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -6,8 +6,8 @@ def knowledge_base_url
     'http://openstax.force.com/support/articles/FAQ/Can-t-log-in-to-your-OpenStax-account'
   end
 
-  def openstax_url
-    'https://openstax.org'
+  def accounts_url
+    main_app.root_path
```
Comment (BryanEliDimas) [inline app/helpers/application_helper.rb]: Isn't this the same as doing `'/'`?

Comment (BryanEliDimas) [reply app/helpers/application_helper.rb]: Which would point to Accounts (`accounts-dev.openstax.org`, `accounts.openstax.org` in prod, etc.) whereas we want it to point to "osweb" (`cms-dev.openstax.org`, `openstax.org` in prod, etc.)

Comment (edwoodward) [reply app/helpers/application_helper.rb]: Possibly. I implemented what was used in the old flow based on a conversation with Brittany.

Comment (edwoodward) [reply app/helpers/application_helper.rb]: Updated to use deployment information for openstax URL

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -6,8 +6,8 @@ def knowledge_base_url
     'http://openstax.force.com/support/articles/FAQ/Can-t-log-in-to-your-OpenStax-account'
   end
 
-  def openstax_url
-    'https://openstax.org'
+  def logo_url
+    Rails.application.secrets.openstax_url
```
Comment (BryanEliDimas) [inline app/helpers/application_helper.rb]: Please add an entry to secrets.yml.example file?

Comment (edwoodward) [reply app/helpers/application_helper.rb]: Done

**Pull request merged**